package.json

```diff
@@ -18,7 +18,8 @@
     "release": "npm run build:release && npm run package:release && npm publish ./package --access public"
   },
   "dependencies": {
-    "@daostack/client": "0.2.6",
+    "@daostack/client": "0.2.67",
```

Comment:
We updated to this version of the client library because the one we are working with right now doesn't accepts `RetryLink`, so I updated it just to test :-)